tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -137,6 +137,10 @@
     <value>Select packages to restore from your backup, clone from repository, or search with Windows Package Manager.</value>
     <comment>Description for Add packages page</comment>
   </data>
+  <data name="AppInstallActivationTitle" xml:space="preserve">
+    <value>App Install Activation</value>

```

Comment:
Should this be sentence case? Other titles in Dev Home are. ("App install activation")

Reply:
Will adjust, thanks for pointing that out!